src/ui/components/label.c

```diff
@@ -39,14 +39,16 @@ typedef struct {
     int16_t text_position_last;
 } data_t;
 
+static void _measure_label_dimensions(component_t* label);
```

Comment:
could also move the function here instead.

Reply:
sure